metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_queries.py

```diff
@@ -225,6 +227,9 @@ def is_allowed_table(self, name: str) -> bool:
     def get_workunits_internal(
         self,
     ) -> Iterable[MetadataWorkUnit]:
+        with self.report.users_fetch_timer:
+            users = self.fetch_users()
```

Comment:
It can be tough to figure out what's in the dictionary (key and value) especially when variables go through different methods. To help, I suggest using a more descriptive name or a type alias to clearly describe the content.

Reply:
looks better now ?

Reply:
yes